website/www/site/content/en/documentation/sdks/python-unrecoverable-errors.md

```diff
@@ -0,0 +1,60 @@
+---
+type: languages
+title: "Unrecoverable Errors in Beam Python"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Unrecoverable Errors in Beam Python
+
+## What is an Unrecoverable Error?
+
+An unrecoverable error is an issue at job start-up time that will
+prevent a job from ever running successfully, usually due to some kind
+of misconfiguration. Solving these issues when they occur is key to
+successfully running a Beam Python pipeline.
+
+## Common Unrecoverable Errors
+
+### Job Submission/Runtime Python Version Mismatch
```

Comment:
If you plan to reference these errors in logs, let's add markdown anchors for better linkability, as titles might change but we can keep the same anchors, so links will be preserved.